metal/loadbalancers.go

```diff
@@ -388,6 +393,12 @@ func (l *loadBalancers) addService(ctx context.Context, svc *v1.Service, ips []p
 			// if we did not find an IP reserved, create a request
 			klog.V(2).Infof("no IP assignment found for %s, requesting", svcName)
 			// create a request
+			// our logic as to where to create the IP:
+			// 1. if metro is set globally, use it; else
+			// 2. if facility is set globally, use it; else
+			// 3. if Service.Metadata.Labels["topology.kubernetes.io/region"] is set, use it; else
+			// 4. if Service.Metadata.Labels["topology.kubernetes.io/zone"] is set, use it; else
```

Comment:
A user is able to offer conflicting annotations vs other settings.
The rules spelled out how this will be processed so I can't complain.

When both Region and Zone are set, especially when the zone resides in the region, I think it's reasonable to assume the more granular, Zone, will be used.  This is not the case today.  Based on how EM EIPs work, I think users supplying both settings will be better served with the Regional EIP (Metro based).